README.md

```diff
@@ -1,126 +1,241 @@
 
-# "parallelizable tests": the minimum test set that should be run
-ctest -j $(nproc) -LE _tests
+#### Parallelizable Tests
+This test suite consits of any test which completes in a short amount of time and does not require shared resources such as file descriptors, specific folders, or ports, and can therefore be run concurrently in different threads without side effects (hence, easily parallelized). These are mostly unit tests and [small tests](https://testing.googleblog.com/2010/12/test-sizes.html).
 
-# Also consider running the WASM spec tests for more coverage
-ctest -j $(nproc) -L wasm_spec_tests
+You can invoke them using `ctest` from a terminal in your Leap build directory.
+```bash
+ctest -j "$(nproc)" -LE _tests
 ```
 
-Some other tests are available and recommended but be aware they can be sensitive to other software running on the same host and they may **SIGKILL** other nodeos instances running on the host.
+#### WASM Spec Tests
+The WASM spec tests verify that our WASM execution engine is compliant with the web assembly standard. These are very [small](https://testing.googleblog.com/2010/12/test-sizes.html), very fast unit tests. However, there are over a thousand of them so the suite can take a little time to run. These tests are extremely CPU-intensive.
+
+You can invoke them using `ctest` from a terminal in your Leap build directory.
+```bash
+ctest -j "$(nproc)" -L wasm_spec_tests
 ```
-cd build
+We have observed severe performance issues when two virtual machines are running this test suite on the same physical host at the same time, for example in a CICD system. This can be resolved by disabling hyperthreading on the host.
+
+#### Serial Tests
+The serial test suite consists of [medium](https://testing.googleblog.com/2010/12/test-sizes.html) component or integration tests that use specific paths, ports, rely on process names, or similar and cannot be run concurrently with other tests. Serial tests can be sensitive to other software running on the same host and they may `SIGKILL` other `nodeos` processes. These tests take a moderate amount of time to complete, but we recommend running them.
 
-# These tests can't run in parallel but are recommended.
+You can invoke them using `ctest` from a terminal in your Leap build directory.
+```bash
 ctest -L "nonparallelizable_tests"
+```
+
+#### Long-Running Tests
+The long-running tests are [medium-to-large](https://testing.googleblog.com/2010/12/test-sizes.html) integration tests that rely on shared resources and take a very long time to run.
 
-# These tests can't run in parallel. They also take a long time to run.
+You can invoke them using `ctest` from a terminal in your Leap build directory.
+```bash
 ctest -L "long_running_tests"
 ```
+
+### Install
+Once you have [built](#build) Leap and [tested](#test) your build, you can install Leap to your system.
+
+Navigate to your Leap build directory and run this command.
+```bash
+sudo make install
```

Comment:
The instructions above do a `make package` which will pop out a deb and/or rpm file. I think somehow we should encourage users to use those before resorting to a `make install`

Reply:
I added instructions to install the binary, including a note that we recommend this route, before the `make install` instructions.